src/main/java/io/beanmapper/core/BeanMatchStore.java

```diff
@@ -277,7 +278,7 @@ private BeanPropertyWrapper dealWithBeanProperty(
                 if (logger.isDebugEnabled()) {
```

Comment:
Volgens mij wordt dit al standaard in de logger.debug gedaan.
![image](https://user-images.githubusercontent.com/105443742/199031182-a1ba9071-e885-456f-9ebf-1a7ee6979ee9.png)

Reply:
Deze check stat er zodat wrapper.getName() en err.getMessage() niet altijd aangeroepen worden. Als het goed is zou dat een kleine performance improvement moeten geven.